### PR TITLE
Fix use-after-move and improve code style on actions

### PR DESCRIPTION
**Pull request by ranisalt**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

- Fix inverted logical conditions
- Avoid usage of private static members
- Remove redundant else statement
- Use fmt for formatting hotkey messages

**Issues addressed:** 
None

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/actions.cpp [hunk 1]
```diff
@@ -218,17 +219,17 @@ bool Actions::registerLuaEvent(Action* event)
...
-				auto result = useItemMap.emplace(*i, std::move(*action));
-				if (!result.second) {
-					std::cout << "[Warning - Actions::registerLuaEvent] Duplicate registered item with id: " << *i << " in range from id: " << v.front() << ", to id: " << v.at(v.size() - 1) << std::endl;
-					continue;
-				}
+		}
+
+		auto v = action->getItemIdRange();
+		for (auto i = v.begin(); i != v.end(); i++) {
+			auto result = useItemMap.emplace(*i, std::move(*action));
```
Comment (ranisalt) [inline src/actions.cpp:226]: By the way, isn't this using `*action` after moving? I just noticed it lol

**Comment by ranisalt** [comment]
OK I noticed this code had a lot of reinvention of wheel such as `v.at(vector.size() - 1)` instead of `v.back()` or iterating over indexes when a range was available and I decided to slip that in. Also fixed several use-after-move instances

File: src/actions.cpp [hunk 2]
```diff
@@ -26,6 +26,7 @@
 #include "game.h"
 #include "pugicast.h"
 #include "spells.h"
+#include <fmt/format.h>
```
Comment (ramon-bernardo) [inline src/actions.cpp:29]: Why use this in each file instead [this](https://github.com/otland/forgottenserver/pull/3430/commits/5a15e2021e4b932144acf848e283ae7e963231f1#diff-2aece0739543e6f81501e211e382a9e264966cdddecea19ed6182e61c2c0e4c5R46)?

Comment (ranisalt) [reply src/actions.cpp:29]: More includes = more compile time

**Review by nekiro — DISMISSED** [review]

File: src/actions.cpp [hunk 3]
```diff
@@ -210,67 +211,37 @@ bool Actions::registerEvent(Event_ptr event, const pugi::xml_node& node)
 bool Actions::registerLuaEvent(Action* event)
 {
 	Action_ptr action{ event };
-	if (action->getItemIdRange().size() > 0) {
-		if (action->getItemIdRange().size() == 1) {
-			auto id = action->getItemIdRange().front();
-			auto result = useItemMap.emplace(id, std::move(*action));
+	if (!action->getItemIdRange().empty()) {
+		const auto& v = action->getItemIdRange();
+		for (auto id : v) {
```
Comment (nekiro) [inline src/actions.cpp:216]: what if we used better names instead of "v"?

Comment (ranisalt) [reply src/actions.cpp]: Absolutely. Renamed to `range`

**Review by nekiro — DISMISSED** [review]
Perfection

**Review by nekiro — APPROVED** [review]

**Pull request merged**